packages/sveltekit/src/server/handle.ts

```diff
@@ -131,12 +131,24 @@ export function sentryHandle(handlerOptions?: SentryHandleOptions): Handle {
 
 async function instrumentHandle(
   { event, resolve }: Parameters<Handle>[0],
-  options: Required<SentryHandleOptions>,
+  options: SentryHandleOptions,
 ): Promise<Response> {
   if (!event.route?.id && !options.handleUnknownRoutes) {
     return resolve(event);
   }
 
+  // caching the result of the version check in `options.injectFetchProxyScript`
+  // to avoid doing the dynamic import on every request
+  if (options.injectFetchProxyScript == null) {
+    try {
+      // @ts-expect-error - the dynamic import is fine here
+      const { VERSION } = await import('@sveltejs/kit');
```

Comment:
l: Does it make sense to cache this somewhere? 🤔

Reply:
We do :) `options.injectFetchProxyScript` will only be set oncea